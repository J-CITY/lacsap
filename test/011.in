
program HelloWorld;
var
    _int, lazy_town_ : integer;
    __r : real;
Begin
  for i:=2 to n do
  begin
    p:=p*i;
  end;
  int := 1;
END.
