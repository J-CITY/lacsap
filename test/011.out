program 1 0 keyword 334
HelloWorld 1 8 identifier 4000
; 1 18 separator 4
var 2 0 keyword 316
_int 3 4 identifier 4001
, 3 8 separator 3
lazy_town_ 3 10 identifier 4002
: 3 21 separator 5
integer 3 23 identifier 4003
; 3 30 separator 4
__r 4 4 identifier 4004
: 4 8 separator 5
real 4 10 identifier 4005
; 4 14 separator 4
Begin 5 0 keyword 324
for 6 2 keyword 320
i 6 6 identifier 4006
:= 6 7 operation 23
2 6 9 literalInt 7000
to 6 11 keyword 314
n 6 14 identifier 4007
do 6 16 keyword 308
begin 7 2 keyword 324
p 8 4 identifier 4008
:= 8 5 operation 23
p 8 7 identifier 4008
* 8 8 operation 0
i 8 9 identifier 4006
; 8 10 separator 4
end 9 2 keyword 315
; 9 5 separator 4
_int 10 2 identifier 4001
:= 10 7 operation 23
1 10 10 literalInt 7001
; 10 11 separator 4
END 11 0 keyword 315
. 11 3 separator 6
